src/main/java/org/synyx/urlaubsverwaltung/application/application/ApplicationForLeavePermissionEvaluator.java

```diff
@@ -55,9 +56,9 @@ static boolean isAllowedToStartCancellationRequest(Application application, Pers
             && !(signedInUser.hasRole(OFFICE) || ((signedInUser.hasRole(BOSS) || isDepartmentHeadOfPerson || isSecondStageAuthorityOfPerson) && signedInUser.hasRole(APPLICATION_CANCEL)));
     }
 
-    static boolean isAllowedToDeclineCancellationRequest(Application application, Person signedInUser) {
+    static boolean isAllowedToDeclineCancellationRequest(Application application, Person signedInUser, boolean isDepartmentHeadOfPerson, boolean isSecondStageAuthorityOfPerson) {
         return application.hasStatus(ALLOWED_CANCELLATION_REQUESTED)
```

Comment:
Im Kommentar zu dem Status steht noch:
    /**
     * Status after the application for leave was allowed but the applicant wants to cancel the own application.
     * The application for leave is at this point not cancelled and can only be approved by OFFICE Role.
     */
Das mit der Office Rolle ist dann wohl nicht mehr korrekt.

Reply:
besprochen